README.md

```diff
@@ -3,7 +3,7 @@
 A toolkit of great libraries to get started building Typelevel apps on JVM, Node.js, and Native! Our very own flavour of the [Scala Toolkit].
 
 ```scala
-//> using lib "org.typelevel::toolkit::0.0.3"
+//> using dep "org.typelevel::toolkit::0.0.5"
```

Comment:
`dep` is new right? I'm afraid this will break/be confusing for anyone who has an old scala-cli for some reason.

Also I wonder if we can just put `latest.release` as the version, let me check that 🤔

Reply:
`dep` is the new one, but it also is going to be the future. The confusion has to happen at some point :P.

Alternatively I could add a remark saying to use `//> using lib "org.typelevel::toolkit::latest.release"`  if you're using an old scala-cli version

Reply:
Fair enough, hopefully we'll have `//> using toolkit` by then :D.